.envrc

```diff
@@ -0,0 +1,28 @@
+layout node
```

Comment:
would you make shellcheck happy and add `# shellcheck shell=bash` as the first line?
```suggestion
# shellcheck shell=bash
layout node
```